src/index.css

```diff
@@ -15,38 +22,62 @@ h1 {
 
 .left-container {
   width: 50%;
+  padding: 20px;
   border: 1px solid #111010;
-  padding: 2%;
-}
-
-.right-container {
-  width: 40%;
-  padding-left: 20px;
+  background-color: #fff;
+  box-shadow: 0 0 10px rgb(0 0 0 / 10%);
 }
 
 .refresh {
   border-radius: 5%;
   border: none;
-  padding: 0 25px;
   cursor: pointer;
+  background-color: #4caf50;
+  color: white;
+  font-size: medium;
+  padding: 10px;
+}
+
+.refresh:hover {
+  background-color: #45a049;
 }
 
 .score-header {
   display: flex;
   justify-content: space-around;
+  align-items: center;
 }
 
 .score-heading {
   font-size: 20px;
+  color: #333;
 }
 
 .score-list {
   list-style: none;
   padding: 0;
+  overflow: auto;
+  max-height: 21vh;
 }
 
 .score-list li {
   margin-bottom: 5px;
+  background-color: #fff;
+  padding: 10px;
+  transition: background-color 0.3s ease;
+  box-shadow: 4px 4px 6px rgb(0 0 0 / 10%), -4px -4px 6px white;
+}
+
+.score-list li:hover {
+  background-color: rgb(255 255 255 / 80%);
+  cursor: pointer;
+}
+
+.right-container {
+  width: 40%;
+  background-color: #fff;
+  box-shadow: 0 0 10px rgb(0 0 0 / 10%);
+  padding: 20px;
```

Comment:
- [REQUIRED] Styling is a student's choice and it does not need to be perfect. You're only required to keep the general layout from the wireframe. Your layout is a bit different from the wireframe. Kindly look into it an d fix.

### Your Design
![your-layout](https://github.com/Abbas-Aljuaifari/LeaderBoard_project/assets/26597432/46c0f7e8-8c63-4e3b-a9de-2b51bc4a6fea)

### Figma Design 
![wireframe](https://github.com/Abbas-Aljuaifari/LeaderBoard_project/assets/26597432/4eea2f62-5fe7-49be-b275-e2eb7e4e4d1a)